src/features/EditMoviePage/EditMoviePage.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { propTypesShapes } from '../../constants';
+import CustomInput from '../../components/CustomInput/CustomInput';
+import CustomTextarea from '../../components/CustomTextarea/CustomTextarea';
+import CustomButton from '../../components/CustomButton/CustomButton';
+import { editMovie } from '../../redux/movies/movies.actions';
+import { moviesDataSelector } from '../../redux/movies/movies.selectors';
+
+import styles from './EditMoviePage.module.scss';
+
+
+const EditMoviePage = ({ match, movies, editMovie, history }) => {
+  const editableMovieId = Number(match.params.id);
+
+  const { id, title, posterUrl, director, genres, description } = movies.find((movie) => movie.id === editableMovieId);
+
+  const INITIAL_DATA = {
+    title,
+    posterUrl,
+    director,
+    genres,
+    description,
+  };
+
+  const [formData, changeFormData] = useState(INITIAL_DATA);
+
+  const [errors, setErrors] = useState([]);
```

Comment:
1. move it to top
2. Do not separate const using empty line, usually we use empty line to separate smth than is totally different by type. Example1: group of constants then empty line, then hook. Example2: methods/handlers